payment/app.py

```diff
@@ -12,39 +12,164 @@
                               password=os.environ['REDIS_PASSWORD'],
                               db=int(os.environ['REDIS_DB']))
 
-
 def close_db_connection():
     db.close()
 
+gateway_url = os.environ['GATEWAY_URL']
 
 atexit.register(close_db_connection)
 
+class User:
+    def __init__(self, user_id: str):
+        self.user_id = user_id
+        self.credit = 0
+    
+    def to_dict(self):
+        return {
+            "user_id" : self.user_id,
+            "credit" : self.credit,
+        }
 
 @app.post('/create_user')
 def create_user():
-    pass
+
+    # Create new user id
+    new_user_id = f"item:{random.getrandbits(ID_BYTES_SIZE)}"
+    while db.hget(new_user_id, "item_id"):
+        new_user_id = f"item:{random.getrandbits(ID_BYTES_SIZE)}"
```

Comment:
Please change the new_user_id to "user:<random_number>" and you should use "user_id" instead of "item_id".